public/app/features/admin/UserAdminPage.tsx

```diff
@@ -61,31 +61,30 @@ export const UserAdminPage = ({
 }: Props) => {
   const { id = '' } = useParams();
   useEffect(() => {
-    const userId = parseInt(id, 10);
-    loadAdminUserPage(userId);
+    loadAdminUserPage(id);
   }, [id, loadAdminUserPage]);
 
   const onPasswordChange = (password: string) => {
     if (user) {
-      setUserPassword(user.id, password);
+      setUserPassword(user.uid, password);
```

Comment:
Do we always expect user UID to be set in the DB? Do you think there are any cases where we need to fall back to using the ID?

Reply:
Yes, there shouldn't be cases without a user UID after the migrations